notebooks/stitches_dev/Notebook6_throwout_Duplicates-across-ensemble-members.Rmd

```diff
@@ -135,6 +148,13 @@ matched_data%>%
   permute_stitching_recipes(N_matches = 5, matched_data = .,
                             archive = archive_subset) ->
   recipes
+
+# TODO 
+# Okay some times the generate_gridded_recepie funtion throws an error related to the neighborhood matching 
+# but it is not consistent. 
```

Comment:
I think that the issue you raised in the other draft PR resolve the issue with this function but I would be interested in hearing if you run into any issues here.